extensions/odoo_theme/static/scss/_sphinx_basic.scss

```
@@ -615,6 +615,11 @@ ul.simple p {
     margin-bottom: 0;
 }
 
+ol > li > ol.simple > li:last-child > p,
+ul > li > ul.simple > li:last-child > p {
```

Hello 👋 
What happens if the element inside the last item is not a paragraph? (e.g. a link, a button, an image, etc.) ?
Not sure it will work 🤔 .

What you could do is apply the margin to the `ul`.

```suggestion
li > .simple {
```

☝️ This way, you'll select all `simple` lists which are in a parent list and apply on it a margin-bottom.

You could go a step further and make the selector more generic. Instead of using a class (here `.simple`) to target lists within another list, you could use this (to be tested) :

```suggestion
li > ul, li > ol {
```

---

Can you tell me how I can test your branch? :)

---

Hi @Brieuc-brd I had to make a small adjustment to my styles, it didnt account for each scenario, it is now:
```
ol.simple > li > p,
ul.simple > li > p,
ol.simple > li > ul > li > p,
ul.simple > li > ul > li > p,
ol.simple > li > ol > li > p,
ul.simple > li > ol > li > p {
    margin-top: 0;
    margin-bottom: 0;
}

ul > li > ul.simple > li:last-child > p,
ol > li > ul.simple > li:last-child > p {
    margin-bottom: 1em;
}
```

You can test this by either pulling down this branch `15.0-scss-sub-bullet-fix-sali` to your local computer by following [these instructions](https://github.com/odoo/documentation/blob/17.0/README.md), or open this branch on runbot to [view the built documentation with these styles](https://runbot207.odoo.com/runbot/static/build/57847964-15-0/logs/build/html/applications/hr/time_off.html#time-off-requests-section).

Once viewing the documentation, navigate to the **Human Resources > Time Off** page and scroll to the **Time off requests section**.  Here you will see a non-simple `ul` with a sub-`ul` that is `.simple`.

Then further down in the document are different variations of these list types.

You can also look at the Table of Contents files to verify that these style changes are not affecting them.

I believe that these style changes fix the issue that this PR aims to address, let me know your thoughts! 
Thanks 🙏